Project/src/MakeCall/CallCard.js

```diff
@@ -111,12 +117,36 @@ export default class CallCard extends React.Component {
                 });
             });
 
+            let localVolumeStateSetter = undefined;
+            let handleSelectedMichrophoneVolumeSubscription = async () => {        
+                let localVolumeIndicator = await (new LocalAudioStream(this.deviceManager.selectedMicrophone).getVolume());
+                localVolumeStateSetter = ()=>{
+                    this.setState({ localVolumeLevel: localVolumeIndicator.level });
+                }
+                localVolumeIndicator.on('levelChanged', localVolumeStateSetter);
```

Comment:
this may be very chatty and cause lots of UI updates, please test it and if needed , debounce UI updates

Reply:
The volume indicator bar transitions are smooth (sampled every 100ms), and things like smoothning constant and sampling time are ECS configurable.

Reply:
y lets debounce to every min or two :)